data/vanilla-0.16.51.json

```diff
@@ -9385,4 +9385,4 @@
             "speed": 0.03125
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
I like this change the most. :)

Reply:
yeah, that was a auto-format, sorry.

Reply:
That was no irony. It messes up diffs at the end of the file.